docs/eventing/sources/ping-source/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "PingSource"
+weight: 31
+type: "docs"
+aliases:
+  - /docs/eventing/sources/pingsource
+  - ../cronjob-source
+  - /docs/eventing/samples/ping-source
+---
+
+![version](https://img.shields.io/badge/API_Version-v1beta2-red?style=flat-square)
+
+A PingSource is an event source that produces events with a fixed payload on a specified [cron](https://en.wikipedia.org/wiki/Cron) schedule.
+
+The following example shows how you can configure a PingSource as an event source that sends events every minute to a Knative service named `event-display` that is used as a sink.
+
+## Before you begin
+
+1. To create a Knative service to use as a sink, you must install [Knative Serving](../../../install).
+1. To create a PingSource, you must install [Knative Eventing](../../../eventing). The PingSource event source type is enabled by default when you install Knative Eventing.
+1. Optional: You can use either `kubectl` or [`kn`](../../../client/install-kn) commands to create components such as a Knative service and PingSource.
+1. Optional: You can use either `kubectl` or [`kail`](https://github.com/boz/kail) for logging during the verification step in this procedure.
+
+## Procedure
+
+1. Optional: Create a new namespace called `pingsource-example` by entering the following command:
+
+    ```shell
+    kubectl create namespace pingsource-example
+    ```
+
+    Creating a namespace for the PingSource example allows you to isolate the components created by this demo, so that it is easier for you to view changes and remove components when you are finished.
+
+1. To verify that the PingSource is working correctly, create a simple Knative service that dumps incoming messages to its log, by entering the command:
+
+    {{< tabs name="create-service" default="kn" >}}
+    {{% tab name="YAML" %}}
+
+```shell
+kubectl -n pingsource-example apply -f - << EOF
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - name: event-display
+          image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+EOF
+```
+
+    {{< /tab >}}
+    {{% tab name="kn" %}}
+
+```shell
+kn service create event-display --image gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+```
+    {{< /tab >}}
+    {{< /tabs >}}
+    <br>
+
+1. Create a PingSource that sends an event containing `{"message": "Hello world!"}` every minute, by entering the command:
+
+    {{< tabs name="create-source" default="kn" >}}
+    {{% tab name="YAML" %}}
+
+```shell
+kubectl -n pingsource-example apply -f - <<EOF
+apiVersion: sources.knative.dev/v1beta2
+kind: PingSource
+metadata:
+  name: test-ping-source
+spec:
+  schedule: "*/1 * * * *"
+  contentType: "application/json"
+  data: '{"message": "Hello world!"}'
+  sink:
+    ref:
+      apiVersion: v1
```

Comment:
If you use Knative Serving, it should be `serving.knative.dev/v1`

Reply:
Removed mentions of serving so not sure if this is still relevant?